Airwallex/Card/Internal/AWXCardViewController.m

```diff
@@ -134,6 +134,77 @@ - (void)viewDidLoad {
         [stackView addArrangedSubview:[self switchOfType:SaveCardSwitch]];
     }
 
+    if (self.viewModel.isBillingInformationRequired) {
+        [stackView addArrangedSubview:self.billingStackView];
+    }
+
+    _confirmButton = [AWXActionButton new];
+    _confirmButton.enabled = YES;
+    [_confirmButton setTitle:NSLocalizedString(@"Confirm", @"Confirm") forState:UIControlStateNormal];
+    [_confirmButton addTarget:self action:@selector(confirmPayment:) forControlEvents:UIControlEventTouchUpInside];
+    [stackView addArrangedSubview:_confirmButton];
+    [_confirmButton.heightAnchor constraintEqualToConstant:52].active = YES;
+
+    if (self.session.billing) {
```

Comment:
I'm even thinking to completely remove session from VC, all those view related display info will refer to the view model instead. But no essentially in this task.